sdk/identity/identity/test/node/managedIdentityCredential.spec.ts

```diff
@@ -20,6 +20,8 @@ interface AuthRequestDetails {
 
 describe("ManagedIdentityCredential", function() {
   afterEach(() => {
+    delete process.env.IDENTITY_ENDPOINT;
+    delete process.env.IDENTITY_HEADER;
```

Comment:
Shouldn't this be restricted to the version of the service they're using? I'm wondering if it would be possible for one of these to be valid in a future version and we delete it.

Reply:
I don't really know! @schaabs if you don't mind

Reply:
Actually, I don't understand the question. What do you mean? In the context of this test.

Reply:
Should be fine in a test case. I had misread it and thought this was a live scenario.